installation/language/de-DE/joomla.cli.ini

```
@@ -0,0 +1,41 @@
+; Author J!German (Joomla! German Translation Team)
+; (C) 2005 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+INSTL_ADMIN_EMAIL_DESC="Die E-Mail-Adresse des Super Benutzers der Website eingeben"
+INSTL_ADMIN_EMAIL_DESC_SHORT="E-Mail-Adresse des Super Benutzers der Website"
+INSTL_ADMIN_PASSWORD_DESC="Das Passwort für den Super Benutzer festlegen"
+INSTL_ADMIN_PASSWORD_DESC_SHORT="Passwort für den Super Benutzer"
+INSTL_ADMIN_USERNAME_DESC="Den Benutzernamen für den Super Benutzer festlegen"
+INSTL_ADMIN_USERNAME_DESC_SHORT="Benutzernamen für den Super Benutzer"
+INSTL_ADMIN_USER_DESC="Den echten Namen des Super Benutzers eingeben"
+INSTL_ADMIN_USER_DESC_SHORT="Echter Namen des Super Benutzers"
+INSTL_DATABASE_COULD_NOT_CONNECT="%s"
+INSTL_DATABASE_ENCRYPTION_CA_LABEL="Pfad zur CA-Datei, anhand derer die Verschlüsselung überprüft werden soll"
+INSTL_DATABASE_ENCRYPTION_CA_LABEL_SHORT="Pfad zur CA-Datei, anhand derer die Verschlüsselung überprüft werden soll"
+INSTL_DATABASE_ENCRYPTION_CERT_LABEL="Pfad zum SSL-Zertifikat für die Datenbankverbindung. Erfordert, dass die Verschlüsselungsoption auf 2 eingestellt ist."
+INSTL_DATABASE_ENCRYPTION_CERT_LABEL_SHORT="Pfad zum SSL-Zertifikat für die Datenbankverbindung. Erfordert, dass die Verschlüsselungsoption auf 2 eingestellt ist."
+INSTL_DATABASE_ENCRYPTION_CIPHER_LABEL="Unterstützte Cipher Suite (optional)"
+INSTL_DATABASE_ENCRYPTION_CIPHER_LABEL_SHORT="Unterstützte Cipher Suite (optional)"
+INSTL_DATABASE_ENCRYPTION_KEY_LABEL="SSL-Schlüssel für die Datenbankverbindung. Erfordert, dass die Verschlüsselungsoption auf 2 gesetzt wird"
+INSTL_DATABASE_ENCRYPTION_KEY_LABEL_SHORT="SSL-Schlüssel für die Datenbankverbindung. Erfordert, dass die Verschlüsselungsoption auf 2 gesetzt wird"
+INSTL_DATABASE_ENCRYPTION_MODE_LABEL="Verschlüsselung für die Datenbankverbindung. Werte: 0=Keine, 1=Einweg, 2=Zwei-Weg"
+INSTL_DATABASE_ENCRYPTION_MODE_LABEL="Verschlüsselung für die Datenbankverbindung. Werte: 0=Keine, 1=Einweg, 2=Zwei-Weg"
+INSTL_DATABASE_ENCRYPTION_VERIFY_SERVER_CERT_LABEL="Das SSL-Zertifikat für die Datenbankverbindung überprüfen.. Werte: 0=Nein, 1=Ja. Erfordert, dass die Verschlüsselungsoption auf 1 oder 2 gesetzt wird."
+INSTL_DATABASE_ENCRYPTION_VERIFY_SERVER_CERT_LABEL_SHORT="Das SSL-Zertifikat für die Datenbankverbindung überprüfen. Werte: 0=Nein, 1=Ja. Erfordert, dass die Verschlüsselungsoption auf 1 oder 2 gesetzt wird."
+INSTL_DATABASE_HOST_DESC="Datenbank-Host"
+INSTL_DATABASE_HOST_DESC_SHORT="Datenbank-Host"
+INSTL_DATABASE_NAME_DESC="Name der Datenbank"
+INSTL_DATABASE_NAME_DESC_SHORT="Name der Datenbank"
+INSTL_DATABASE_PASSWORD_DESC="Passwort des Datenbanknutzers"
```

```suggestion
INSTL_DATABASE_PASSWORD_DESC="Datenbank Passwort"
```
Siehe https://github.com/joomlagerman/joomla/blob/9cb11a7a5df219bd2780f5a3e3b1019cdd016a12/installation/language/de-DE/joomla.ini#L92